parkings/importers/__init__.py

```diff
@@ -1 +1,2 @@
 from .parking_areas import ParkingAreaImporter  # noqa
+from .permit_areas import PermitAreaImporter  # noqa
```

Comment:
The correct way to do this is: ```
from .parking_areas import ParkingAreaImporter
from .permit_areas import PermitAreaImporter
__all__ = [
    'ParkingAreaImporter',
    'PermitAreaImporter',
]
```
so you don't need to add these nasty `# noqa` comments.